src/material-examples/cdk-datepicker-disabled/cdk-datepicker-disabled-example.ts

```diff
@@ -0,0 +1,24 @@
+import {Component} from '@angular/core';
+import {CalendarView} from '@angular/cdk/datetime';
+
+/** @title CDK Disabled datepicker */
```

Comment:
Is `disabled` a complex enough use-case that it warrants its own example? I would think that you'd just need one main example of custom calendar with `CdkDatepicker` that has options for disabling and could show a snackbar or similar when there's an event.

Reply:
The custom calendar doesn't have access to `CdkDatepicker` and whether or not the datepicker gets disabled